tests/Order/Customer/LocaleTest.php

```diff
@@ -31,7 +31,10 @@ public function itCanBeEmptyString()
     public function itShouldFailOnIncorrectLocale()
     {
         $this->setExpectedException('Assert\InvalidArgumentException');
-        Locale::fromString('en_US');
+        Locale::fromString('en');
```

Comment:
I still don't understand why this passes the test. You'd think that 'en' is a valid locale, so the InvalidArgumentException should not be thrown. I think you need to split up this test into three separate ones.